src/server_manager/install_scripts/install_server.sh

```diff
@@ -378,14 +378,18 @@ install_shadowbox() {
   readonly SB_IMAGE=${SB_IMAGE:-quay.io/outline/shadowbox:stable}
 
   log_for_sentry "Setting PUBLIC_HOSTNAME"
-  # TODO(fortuna): Make sure this is IPv4
   PUBLIC_HOSTNAME=${FLAGS_HOSTNAME:-${SB_PUBLIC_IP:-$(curl -4s https://ipinfo.io/ip)}}
 
   if [[ -z $PUBLIC_HOSTNAME ]]; then
     local readonly MSG="Failed to determine the server's IP address."
     log_error "$MSG"
     log_for_sentry "$MSG"
     exit 1
+  elif ! getent ahosts $PUBLIC_HOSTNAME > /dev/null; then
```

Comment:
This seems like a very obscure command. I don't actually know what it's doing, the documentation makes it sound like it's for something else. Also, how well supported is that?

With the explicit check for curl success and the --fail flag, I think we are good. ipinfo will always return a valid IP on success. If you really want to check the IP, perhaps something like `[[ $PUBLIC_IP =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$ ]]` will be clearer. (note the use of PUBLIC_IP, not PUBLIC_HOSTNAME.

Reply:
OK, removed.